### DISCUSSION TITLE
Architecture Community [General Information]

### DISCUSSION THREAD

**Discussion by antonkri** [eclipse-score/score#110] [Architecture Community]
## Members
   - Lead: @qor-lb 
   - Proxy: @arsibo 

## Regular Meetings on [Eclispe SDV Calendar](https://calendar.google.com/calendar/newembed?src=c_2ampi2bmoka3qter4dceap1d5g@group.calendar.google.com&ctz=Europe/Berlin)
  - Name: [sdv-wg-SCORE] Architecture Community
  - Frequence:    weekly
  - Minutes: https://github.com/orgs/eclipse-score/discussions/2457
 
## Slack Channel
   - [#score-architecture-community](https://sdvworkinggroup.slack.com/archives/C08C1HG5AKY)

## Team Plan
   - https://github.com/orgs/eclipse-score/projects/41

## Repositories
  - https://github.com/eclipse-score/score

**Comment by antonkri** [comment]
@qor-lb please add link to slack channel

**Reply by qor-lb** [reply]
done 👍

**Reply by antonkri** [reply]
Thanks!
